packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -189,13 +195,14 @@ export class HbarLimitService implements IHbarLimitService {
       spendingPlan = await this.createBasicSpendingPlan(ethAddress, ipAddress);
     }
 
-    const dailyLimit = HbarLimitService.DAILY_LIMITS[spendingPlan.subscriptionType];
+    const dailyLimit = HbarLimitService.TIER_LIMITS[spendingPlan.subscriptionType].toTinybars();
```

Comment:
Still says daily.
Is it?

Reply:
Oops, good catch. I will update the name of this variable.